resources/resource_transformers/tocss/dartsass/integration_test.go

```diff
@@ -271,3 +271,35 @@ T1: {{ $r.Content }}
 	})
 
 }
+
+func TestOptionVars(t *testing.T) {
+	if !dartsass.Supports() {
+		t.Skip()
+	}
+
+	files := `
+-- assets/scss/main.scss --
+p {
+	color: $color1;
+	font-size: $font_size;
+}
+b {
+	color: $color2;
+}
+-- config.toml --
+-- layouts/index.html --
+{{ $vars := dict "$color1" "blue" "$color2" "green" "font_size" "24px" }}
+{{ $cssOpts := (dict "transpiler" "dartsass" "outputStyle" "compressed" "vars" $vars ) }}
```

Comment:
@jmooring this is a little rough around the edges, but I think this will be the API.